themes/digital.gov/layouts/shortcodes/note.html

```diff
@@ -38,7 +38,7 @@
 
 */}}
 
-<div class="note {{ if .Get 0 -}}{{- .Get 0 -}}{{- end -}}">
```

Comment:
Curious what is `0` in this context?

Reply:
Are there any cards in the backlog we can add a note to refactor it for clarity?

Mainly, setting via a `modifier` parameter instead of position.

Reply:
Seems fair, not many short codes use positional parameters as they can be unclear and confusing in general. I've created a card for updating this to a named parameter.